installation/resources/installer-config-gke-integration.yaml.tpl

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
from where were these set before you added https://github.com/kyma-project/test-infra/pull/4477/files ? I dont see anything removed in these PRs ?

Reply:
they are not removed in the other PR for backwards compatibility